arches_modular_reports/app/views/modular_report.py

```diff
@@ -42,12 +42,14 @@
 @method_decorator(can_read_resource_instance, name="dispatch")
 class ModularReportConfigView(View):
     def get(self, request):
-        """Just get first. But if there are multiple in the future,
-        the vue component will need to know which one to request."""
+        filters = Q(graph__resourceinstance=request.GET.get("resourceId"))
+
+        report = request.GET.get("reportConfigName", None)
+        if report:
+            filters &= Q(config__name__iexact=report)
```

Comment:
Perfect. This has the advantage of being future-proof against future versions of Django that change how filtering against null k/v pairs work.